.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,8 +21,11 @@ Please be as specific as possible and give set-up details.
 - [ ] container (Kubernetes, Docker, containerd, etc. please specify)
 - [ ] or a combination, please be explicit
 - [ ] jails if it is FreeBSD
+- [ ] classic packaging (rpm, deb as per OS)
```

Comment:
I feel like this is a bit confusing since onedir packaging also provides rpm,debs per OS

Reply:
I know, that is why I added the rpm, deb as per OS.
Open to suggestions, but we have been referring to it as classic, how about **_regular packaging_**, that is,
```
- [ ] regular packaging
```

Reply:
I'm fine with the naming `classic packaging`. I am suggesting to remove this `(rpm, deb as per OS)` since onedir also provides rpm's and deb's per OS

Reply:
Done removed  ```(rpm, deb as per OS)```